docs/source/tutorials_source/package/tutorial_simclr_clothing.py

```diff
@@ -290,18 +314,15 @@ def plot_knn_examples(embeddings, n_neighbors=3, num_examples=6):
 torch.save(state_dict, 'model.pth')
 
 # %%
-# THIS COULD BE IN A NEW FILE (e.g. inference.py
+# THIS COULD BE IN A NEW FILE (e.g. inference.py)
 #
 # Make sure you place the `model.pth` file in the same folder as this code
 
 # load the model in a new file for inference
 resnet18_new = torchvision.models.resnet18()
-last_conv_channels = list(resnet.children())[-1].in_features
+
 # note that we need to create exactly the same backbone in order to load the weights
-backbone_new = nn.Sequential(
-    *list(resnet.children())[:-1],
-    nn.Conv2d(last_conv_channels, num_ftrs, 1),
-)
+backbone_new = nn.Sequential(*list(resnet18_new.children())[:-1])
```

Comment:
Why is the `nn.Conv2d(last_conv_channels, num_ftrs, 1)` not needed anymore?

Reply:
It is not part of the original SimCLR architecture and the network works without it.